src/components/page/page.scss

```diff
@@ -1,3 +1,12 @@
+html {
```

Comment:
do not match tags. BTW if you need app to be full screen width -- just set .app's height to `100vh` -- and there will be no need to set height 100% on html, page and main-container